static/js/place/place_template.tsx

```diff
@@ -411,10 +445,13 @@ class ChildPlace extends Component<ChildPlacePropType, {}> {
     }
     return (
       <React.Fragment>
+        <span id="child-place-head">Places in {this.props.placeName}</span>
         {Object.keys(this.props.childPlaces).map((placeType) => (
-          <div key={placeType}>
-            <div className="child-place-type">{pluralize(placeType)}</div>
-            {this.props.childPlaces[placeType].map((place, i) => (
+          <div key={placeType} className="child-place-group">
+            <div className="child-place-type">{pluralizedDisplayNameForPlaceType(placeType)}</div>
+            {this.props.childPlaces[placeType]
+            .sort((a, b) => a.name > b.name)
```

Comment:
can remove this sort() as input as already sorted

Reply:
It seems to be sorted by population (maybe it's done in the mixer, but it's not in the website repo), but I think alphabetical is better (easier to scan and find a child place).